content/includes/installation/enable-agent-selinux.md

```diff
@@ -0,0 +1,31 @@
+---
+docs:
+---
+
+The following SELinux files are added when you install the NGINX Agent package:
+
+- `/usr/share/selinux/packages/nginx_agent.pp` - loadable binary policy module
+- `/usr/share/selinux/devel/include/contrib/nginx_agent.if` - interface definitions file
+- `/usr/share/man/man8/nginx_agent_selinux.8.gz` - policy man page
+
+To load the NGINX Agent policy, run:
+
+```bash
+sudo semodule -n -i /usr/share/selinux/packages/nginx_agent.pp
+sudo /usr/sbin/load_policy
+sudo restorecon -R /usr/bin/nginx-agent
+sudo restorecon -R /var/log/nginx-agent
+sudo restorecon -R /etc/nginx-agent
+```
+
+### Add ports to NGINX Agent SELinux context
```

Comment:
Headings inside includes dont populate the Table of contents navigation.

Usually we get around this by not having headings inside includes ( create 1 include for each section) and add the headings to the document before the include shortcode